src/storageMaps/constants.ts

```diff
@@ -48,10 +48,12 @@ export enum OffloadPlugin {
 }
 
 // Reference: https://github.com/kubev2v/forklift/blob/29b60e21c388420e6e0e79a55802c2c0a201ab45/pkg/apis/forklift/v1beta1/mapping.go
```

Comment:
So we need to update this enum whenever the BE changes it?

Reply:
You ask a good question here. There was a scenario where if hypothetically they change the name of a currently defined/known enum for some reason, that would lead to an option in our dropdown with a friendly name that maps to the wrong enum. So I've updated the logic to achieve the following:

- Known values from our enum get nice friendly names
- Unknown values from the API just show as-is
- We don't need to update our enum immediately when the API changes
- We can add friendly names for new values at our own pace